src/app.js

```diff
@@ -1 +1,32 @@
 'use strict';
+
+const readline = require('readline');
+const { checkInputNumbers } = require('./logic.js');
+const { randomNum } = require('./uniqueNum.js');
```

Comment:
It's better to name the files the same as the modules they return:
```
require('./checkInputNumbers')
require('./randomNum')
```
Of course, if you haven just one export at each one.

Reply:
thanks, got it